Foundation/NSError.swift

```diff
@@ -288,7 +288,8 @@ public extension Error where Self : CustomNSError {
 public extension Error {
     /// Retrieve the localized description for this error.
     var localizedDescription: String {
-        return NSError(domain: _domain, code: _code, userInfo: nil).localizedDescription
+        let defaultUserInfo = _swift_Foundation_getErrorDefaultUserInfo(self) as! [String : Any]
```

Comment:
I'm pretty sure I'm seeing a crash directly related to this. My process is crashing when I try to print an error, and the crash stack has swift_Foundation_getErrorDefaultUserInfo at the top...